src/test/java/pazone/ashot/coordinates/WebDriverCoordsProviderTest.java

```diff
@@ -19,7 +20,8 @@ void testGetCoordinatesOfElement() {
         int y = 2;
         int height = 3;
         int width = 4;
-        when(element.getRect()).thenReturn(new Rectangle(x, y, height, width));
+        when(element.getLocation()).thenReturn(new Point(x, y));
+        when(element.getSize()).thenReturn(new Dimension(4, 3));
```

Comment:
please use variables `height` and `width`

Reply:
A stupid oversight. Thanks for the review, gonna fix it.